Common/syntheaPostInstall.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+mkdir /tmp/loader
+pushd /tmp/loader
+curl -fsSL --progress-bar https://github.com/OSEHRA/VistA-FHIR-Data-Loader/releases/download/0.1/VISTA_SYNTHETIC_DATA_LOADER_BUNDLE_0P2T1.KID.zip -o loader.zip
+unzip -q loader.zip -d /tmp/loader/Synthea
+echo "Installing Synthea ingestor patch"
+system=1
+instanceName='-cn $instance'
+if ($installgtm || $installYottaDB); then
+  system=2
+  instanceName=''
+fi
+if [ ! -d "VistA" ]; then
+  echo "Downloading OSEHRA VistA Tester Repo"
+  curl -fsSL --progress-bar https://github.com/OSEHRA/VistA/archive/master.zip -o VistA-master.zip
+  unzip -q VistA-master.zip
+  rm VistA-master.zip
+  mv VistA-master VistA
+fi
+pushd VistA/Scripts/
+# Installs as the DUZ=1 user
+python PatchSequenceApply.py -S $system -p /tmp/loader/Synthea $instanceName -l /tmp/ -i -n ALL -d 1
+popd
```

Comment:
maybe we should popd before we pushd to VistA/Scripts?

Reply:
Looks okay to me; but I will try it.

Reply:
i'm not saying what is written won't work, just seems messy.